mlflow/sagemaker/__init__.py

```diff
@@ -1019,12 +1019,16 @@ def push_model_to_sagemaker(
     _logger.info("Created Sagemaker model with arn: %s", model_response["ModelArn"])
 
 
-def run_local(model_uri, port=5000, image=DEFAULT_IMAGE_NAME, flavor=None):
+def run_local(name, model_uri, flavor=None, config=None):  # pylint: disable=unused-argument
```

Comment:
Make `run_local` conform to the interface required by `mlflow deployments run-local -t sagemaker`

Reply:
These changes are tested by test cases that call `score_model_in_sagemaker_docker_container`, such as https://github.com/mlflow/mlflow/blob/8098785831a3414bed62ec3239b60a6a500f4dbd/tests/spark/test_spark_model_export.py#L292-L297